docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -7,4 +12,342 @@
+
+At the beginning of the co-simulation, the distributions defined above will be sampled $N$ times within the EV federate, where $N =$ the number of parking spots/charging ports in the garage. The output of the initial sampling is the number of requested Level 1, 2, and 3 charging ports. The soc for the batteries on board are initialized to a uniform random number between 0.05 and 0.5, and these SOC are sent to the Charger federate. If the SOC of an EV battery is less than 0.9, the EV Controller federate tells the EV battery in the EV federate to continue charging. Otherwise, the EV Charger disconnects the EV battery, and instructs the it to sample a new EV battery from the distributions (one sample).
+
+After the two federates pass information between each other -- EV Battery sends SOC, EV Charger instructs whether to keep charging or resample the distributions -- the EV Battery calculates the total power demanded in the last time interval.
+
+## Execution and Results
+
+Execution can be done with either a simple script (provided on the repo), or with Merlin.
+
+### Manual Orchestration Execution
+
+Manual implementation of the co-simulation is done with the helper script `make_samples_manual.py`, with command line execution:
+
+```shell session
+$ python make_samples.py
+```
+
+This implementation will run a default co-simulation. The default parameters are:
+
+```python
+    samples = 30
+    output_path = os.getcwd()
+    numEVs = 10
+    hours = 24
+    plot = 0
+    run = 1
+```
+
+This means that we are generating 30 JSON files with unique seeds, we are using the current operating directory as the head for the output path, we are simulating 10 EVs in the co-simulation for one day, we are not running individual plots for each simulation, and we are executing the JSON files with `helics_cli` after they have been created.
+
+If we wanted to run a Monte Carlo co-sim with different parameters, this would be:
+
+```shell session
+$ python make_samples.py 10 . 100 24*7 0 0
+```
+
+This execution would create 10 JSON files with unique seeds, set the current directory as the head for the output path, simulate 100 EVs for a week, not generate plots with each simulation, and not execution JSONs with `helics_cli`.
+
+You may decide to adapt `make_samples_manual.py` to suite your needs within the Merlin environment, in which case you would only need the helper script to create the JSON files. If you elect to execute the JSONs using the helper script, sub directories are created for the `helics_cli` runner JSONs and for the csv results. Results for the default simulation are on the repo and can be used for confirming accurate execution.
+
+```python
+    out_json = output_path+'/cli_runner_scripts'
+    out_data = output_path+'/results'
+```
+
+In the runner scripts directory, there will be 30 JSONs. Each will have a unique `seed` parameter, otherwise they will all look identical:
```

Comment:
```suggestion
In the runner scripts directory, there will be 30 JSON files. Each will have a unique `seed` parameter, otherwise they will all look identical:
```